.travis.yml

```diff
@@ -2,15 +2,11 @@ language: python
 python:
     - "3.5"
     - "3.4"
-before_install:
-    - git clone https://github.com/ONSdigital/sdx-common.git
-    - pip3 install ./sdx-common
 install:
-    - pip install -r requirements.txt
-    - pip install -r test_requirements.txt
+    - make build
     - pip install codecov
 script:
-    - flake8
+    - make test
     - python -m unittest tests/test_service.py
     - coverage run -m unittest tests/*.py
```

Comment:
The `make test` already runs the unit test step - still require the coverage though